changelog.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## `v3.3.13`
```

Comment:
Can you update [version.go](https://github.com/Azure/azure-event-hubs-go/blob/master/version.go) with this version info?

Reply:
Done.